spec/motions-spec.coffee

```diff
@@ -1735,6 +1735,25 @@ describe "Motions", ->
       keydown(',')
       expect(editor.getCursorScreenPosition()).toEqual [0, 2]
 
+    it "gets state from GlobalVimState", ->
+      keydown('f')
+      normalModeInputKeydown('c')
+      expect(editor.getCursorScreenPosition()).toEqual [0, 2]
+
+      # store the current find for 'c'
+      findFC = vimState.globalVimState.currentFind
```

Comment:
Our test helpers are a bit awkward right now for creating other editors. I'd like to simplify them.